traversal/planner/GraphPlanner.java

```diff
@@ -360,15 +341,15 @@ private void optimise(GraphManager graph, boolean singleUse) {
             return;
         }
 
+        if (!isPlanned()) setInitialValues();
```

Comment:
we only need to run the initialiser once, for the first time. In the future, when the statistics change, we change the objective weighting but the previous solution is still a valid solution (though it may not be optimal).
In my mind the narrative is clearer: if not planned at all yet, set some initial values. Otherwise, continue where you left off, after possibly updating the statistics.